types/node/stream/consumers.d.ts

```diff
@@ -0,0 +1,25 @@
+// Duplicates of interface in lib.dom.ts.
+// Duplicated here rather than referencing lib.dom.ts because doing so causes lib.dom.ts to be loaded for "test-all"
+// Which in turn causes tests to pass that shouldn't pass.
+//
+// This interface is not, and should not be, exported.
+interface Blob {
+    readonly size: number;
+    readonly type: string;
+    arrayBuffer(): Promise<ArrayBuffer>;
+    slice(start?: number, end?: number, contentType?: string): Blob;
+    stream(): NodeJS.ReadableStream;
+    text(): Promise<string>;
+}
```

Comment:
I'm not sure this comment is accurate? It looks like this interface is leaking into global definitions:

<img width="944" alt="149748342-48857820-a09b-439a-a7c7-f954d02f1aab" src="https://user-images.githubusercontent.com/12036746/150119460-627813d2-ef19-4228-80bf-3db24ca4e0d1.png">

I can't remember where I read it (maybe in a linter rule or something?), but I'm pretty sure _all_ interfaces in a `.d.ts` file are exported, so that consumers have access to references to them. Very happy to be proven wrong on this point, though.

Reply:
Good news, the correct types have been added in the interim so this erroneous "duplicate" is no longer needed.  I just opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59905 which I think will resolve this error.

Reply:
@thw0rted With the benefit of the doubt I will think you didn't mean it as such but I feel called out here anyway, and so I feel a need to defend myself.

I specifically did *not* add myself to the contributors list for node types because I have no interest in being pinged for reviews and I have hardly any interest in Node's types to begin with as long as they work, and this has for many months. 

Furthermore, even though yes it might be wrong that I made this change, that change was made in a pull request just like your own. That means it was reviewed and it was merged by a DT maintainer. Simply denotsting that I did bad here is therefore coming over as shallow because clearly other people were involved as well.

Reply:
You're right that I didn't mean it as a personal attack -- reading back through my comment I can definitely see how you'd take it that way, and I'm sorry.  (I can sympathize about not signing yourself up as a contributor, the scope is unlike anything else on DT!)

The situation is very confusing, and I don't know that anybody can really be blamed directly -- Node chose to add this partial implementation of some web APIs, and worse they're reusing web class names for incomplete implementations that aren't available at the same scope.  When you made your PR you didn't even have a "Node Blob" to reference, so your approach was totally reasonable; bleeding into global was an honest mistake.

To make matters worse, I don't see any way to write a test (or I guess a "meta" test?) that would prevent this.  Maybe the package.json or tsconfig file could have a field with a list of globals introduced by the package, and if anything else bleeds out, the test would fail?  I don't really know enough about TS internals to suggest how to implement something like that but it might be worth kicking the idea around in Discussions -- I'll take a look.